### ISSUE TITLE
Reflow navigation bar colors in high contrast mode

### ISSUE THREAD

**Issue by lisli1** [microsoft/accessibility-insights-web#2914]
## Describe the bug

**HC issue 1:** 
Color contrast issue in navigation bar in high contrast mode when hovering a selected nav item (black text against dark gray background). Hover color should be same as in Production Version 2.18.2.

**HC issue 2:** 
Selected test in FastPass has light blue color #c7e0f4 in high contrast mode. The selected test in FastPass should have the same background color as is currently in Production Version 2.18.2.

**To Reproduce**

1. Go to any target page
2. Open the extension and select "Assessment" from the popup
3. Select "Settings" from the settings gear dropdown on the top right corner
4. Toggle "Enable high contrast" to "On"
5. Hover over the selected item in the navigation bar to see color contrast issue (HC issue 1)
6. On the switcher, select FastPass to see color discrepancy (HC issue 2)

## Expected behavior

**HC issue 1:** 
Hover on selected nav item should have sufficient color contrast between the text and background. Hover color should be same as in Production Version 2.18.2 (light gray on nav items, no hover color on selected item).

**HC issue 2:** 
The selected test in FastPass should have the same background color as is currently in Production Version 2.18.2. Highlight only the selected requirement, not the expanded test group.

## Screenshots

**HC issue 1:** Screenshot of Visible focus requirement selected and hovered, with black text against dark gray background, while in HC mode

![selected-hovered-color-contrast](https://user-images.githubusercontent.com/48259897/84987235-a251cc80-b0f4-11ea-986e-202dec101754.png)

**HC issue 2:** Screenshot of FastPass with Automated checks selected in Production (left) and in Canary (right), while in HC mode

![fastpass-automated-checks-styles](https://user-images.githubusercontent.com/48259897/84989996-5c4b3780-b0f9-11ea-84f9-6a61b5dea0f9.png)

## Context (please complete the following information)

- OS Name & Version:  Windows 10.0.18363 Build 18363
- AI-Web Version & Environment:  Canary Version 2020.6.17.2339
- Browser Version:  Microsoft Edge Version 83.0.478.45 (Official build) (64-bit)
- Target Page: any

**Comment by ferBonnin** [comment]
Per conversation with Peter, we will address this during feature work for the last feature for reflow.
cc @pownkel
